ui/packages/catalog-ui-search/src/main/webapp/js/cql.spec.js

```diff
@@ -130,7 +130,7 @@ describe('tokenize', () => {
         value: '**%%__??\\*\\*\\?\\?',
       }
       const result = cql.write(filter)
-      expect(result).equals("anyText ILIKE '%%\\%\\%\\_\\___\\*\\*\\?\\?'")
+      expect(result).equals('"anyText" ILIKE \'%%\\%\\%\\_\\___\\*\\*\\?\\?\'')
```

Comment:
Does string interpolation work here? It would make things more readable

Reply:
This looks like just a literal value. The expected result from a test. So I don't think there's anything dynamic to interpolate here.

Reply:
oh whoops, I meant using template strings https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
Backslashes still serve as the escape character in template strings, and we're testing backslashes here so at most we're going to save two escapes, both `\'` chars. Not a drastic improvement.